src/lib.rs

```diff
@@ -25,3 +27,17 @@ extern "C" {
 pub unsafe fn init() {
     __afl_manual_init();
 }
+
+pub fn fuzz_run_str<F>(some_closure: F)
+    where F: Fn(&str)
```

Comment:
well, my current thinking is to have `fuzz_run_str` *and* `fuzz_run_bytes`, but I have spent very little time thinking about it. I agree that if it were just one function, it should work on bytes.